community.md

```diff
@@ -48,33 +48,22 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 <input class="cTextfieldstyle" maxlength="90" value="" id="emailUserOS" name="email" placeholder="I consent to join the email list" title="Email" type="text">
 </form> -->
 
-## Want to report issues? 
+## How you can help 

```

Comment:
Shall we try and keep these headings catchy/lively? How about something like "Help us grow" or "We need your help!"

Reply:
Changed as follows:
- Seek for help
- Help us grow